pkg/ironicconductor/volumes.go

```diff
@@ -13,9 +12,6 @@ import (
 func GetVolumes(instance *ironicv1.IronicConductor) []corev1.Volume {
 	var config0640AccessMode int32 = 0640
 	pvcName := fmt.Sprintf("%s-%s", ironic.ServiceName, ironic.ConductorComponent)
```

Comment:
Something is not quite lined up... 

```
status:
  conditions:
  - lastProbeTime: null
    lastTransitionTime: "2024-06-18T08:13:28Z"
    message: '0/3 nodes are available: persistentvolumeclaim "ironic-conductor" not
      found. preemption: 0/3 nodes are available: 3 Preemption is not helpful for
      scheduling..'
```

```
[zuul@controller-0 ~]$ oc get pvc ironic-conductor-ironic-conductor-0 
NAME                                  STATUS    VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS    AGE
ironic-conductor-ironic-conductor-0   Pending                                      local-storage   34m
```

Reply:
I think I figured this out. 
Not sure how to update this PR, so I created a new one: https://github.com/openstack-k8s-operators/ironic-operator/pull/439

The result looks like this after scaling from 1 to 3 replicas:
```
var-lib-ironic-ironic-conductor-0                          Bound     local-storage10-master-1   10Gi       RWO,ROX,RWX    local-storage   5m16s
var-lib-ironic-ironic-conductor-1                          Bound     local-storage03-master-2   10Gi       RWO,ROX,RWX    local-storage   35s
var-lib-ironic-ironic-conductor-2                          Bound     local-storage03-master-0   10Gi       RWO,ROX,RWX    local-storage   4s
```

```
[zuul@controller-0 ~]$ oc get ironicconductors.ironic.openstack.org ironic-conductor 
NAME               NETWORKATTACHMENTS   STATUS   MESSAGE
ironic-conductor   ["ironic"]           True     Setup complete
```

```
[zuul@controller-0 ~]$ oc get pod ironic-conductor-0 -o json | jq .spec.volumes[0]
{
  "name": "var-lib-ironic",
  "persistentVolumeClaim": {
    "claimName": "var-lib-ironic-ironic-conductor-0"
  }
}
[zuul@controller-0 ~]$ oc get pod ironic-conductor-1 -o json | jq .spec.volumes[0]
{
  "name": "var-lib-ironic",
  "persistentVolumeClaim": {
    "claimName": "var-lib-ironic-ironic-conductor-1"
  }
}
[zuul@controller-0 ~]$ oc get pod ironic-conductor-2 -o json | jq .spec.volumes[0]
{
  "name": "var-lib-ironic",
  "persistentVolumeClaim": {
    "claimName": "var-lib-ironic-ironic-conductor-2"
  }
}
```

volumeMounts for containers has:
```
      {
        "mountPath": "/var/lib/ironic",
        "name": "var-lib-ironic"
      },
```

Reply:
I pulled in the changes in https://github.com/openstack-k8s-operators/ironic-operator/pull/439, thanks